pkgs/development/compilers/gcc/11/default.nix

```
@@ -163,10 +163,15 @@ stdenv.mkDerivation ({
   inherit noSysDirs staticCompiler crossStageStatic
     libcCross crossMingw;
 
+  strictDeps = true;
+
   depsBuildBuild = [ buildPackages.stdenv.cc ];
   nativeBuildInputs = [ texinfo which gettext ]
     ++ (optional (perl != null) perl)
     ++ (optional langAda gnatboot)
+    # The builder relies on GNU sed (for instance, Darwin's `sed' fails with
+    # "-i may not be used with stdin"), and `stdenvNative' doesn't provide it.
+    ++ (optional hostPlatform.isDarwin gnused)
```

```suggestion
    ++ optional hostPlatform.isDarwin gnused
```
all those are optional

---

```suggestion
    ++ optional buildPlatform.isDarwin gnused
```

The comment says it will be run by the builder; supposedly not by the produced compiler.
I suppose darwin -> linux cross was broken by this.